src/postPage.ejs

```diff
@@ -0,0 +1,356 @@
+                    "./icons/svgs/login.svg" ).default%>"/> یاد داشت ها
+                  </a>
+                </li>
+                <li class="header-navbar__pofile-menu-item">
+                  <a class="header-navbar__profile-link">
+                    <img class="header-navbar__profile-icon" src="<%=require(
+                    "./icons/svgs/login.svg" ).default%>"/> بازدید های اخیر
+                  </a>
+                </li>
+                <li class="header-navbar__pofile-menu-item">
+                  <a class="header-navbar__profile-link">
+                    <img class="header-navbar__profile-icon" src="<%=require(
+                    "./icons/svgs/login.svg" ).default%>"/> دیوار برای کسب و کار
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </button>
+
+          <button class="header-navbar__button button button--icon">
+            <i class="button__logo">
+              <img src="<%=require( "./icons/svgs/chat-o.svg" ).default%>"/>
+            </i>
+            <span class="button__title"> چت </span>
+          </button>
+
+          <button class="header-navbar__button button">
+            <span class="button__title"> پشتیبانی </span>
+          </button>
+
+          <button class="header-navbar__button button button--primary">
+            <span class="button__title"> ثبت آگهی </span>
+          </button>
+        </nav>
+      </header>
+
+      <div class="single-post-container">
+
+        <section class="breadcrump">
+          <ol class="breadcrump__list">
+          </ol>
+        </section>
+
+        <main class="product">
+            <div class="product__info">
+                <section class="post-actions">
+                    <button class="post-actions__info-button button button--primary">
+                      <span class="button__title">
+                        اطلاعات تماس
+                      </span>
+                    </button>
+                    <div class="post-actions__container">
+                        <button class="post-actions__share-button button button button--circular">
+                            <img class="button__logo" src="<%=require( "/src/icons/svgs/bookmark-o.svg" ).default%>" />
+                        </button>
+                        <button class="post-actions__save-button button button--circular">
+                            <img class="button__logo" src="<%=require( "/src/icons/svgs/share-variant-o.svg" ).default%>" />
+                        </button>
+                    </div>
+                </section>
+                <section class="group-info">
+                  <ul class="group-info__list">
+                  </ul>
+                </section>
+                <section class="pair-info">
+                    <ul class="pair-info__list">
+                    </ul>
+                </section>
+                <section class="post-description">
+                    <h1 class="post-description__title">
```

Comment:
h1?

Reply:
Is the product name in its own page not a heading? and whats more the main heading?

Reply:
ohhh i see the entire file now
yep this smells sorry

Reply:
never mind i saw the whole file
my bad its supposed to be an h2